upgrade_tests/paging_test.py

```diff
@@ -188,6 +188,7 @@ def random_txt(text):
             assert_lists_equal_ignoring_order(pf.all_data(), expected_data, sort_key='value')
 
 
+@pytest.mark.upgrade_test
```

Comment:
Thank you for pointing it. Actually marking `BasePagingTester` is not needed, since it inherits form `UpgradeTester`, which is already marked. My main concern is that the marking becomes implicit. I don't know if it will be confusing or not, while  believe developers will have intention that all tests are marked as upgrade tests in this folder. I can try to clarify in `upgrade_tests/README.md` that using `UpgradeTester` is enough.

I tested that it works in `pytest`, however the readme instructions use `nosetests` to run upgrade tests.

Reply:
I think that the note in the readme makes this clearer, thanks. At some point we will have to clear the references to nosetests, but I think there is no need to do it during this ticket.